dependencies/src/test/java/com/google/cloud/tools/opensource/classpath/ClassDumperTest.java

```diff
@@ -100,11 +100,11 @@ public void testListInnerClasses() throws IOException {
   }
 
   @Test
-  public void testCreationInvalidInput() {
+  public void testCreationInvalidInput() throws IOException {
     try {
       ClassDumper.create(ImmutableList.of(Paths.get("")));
       Assert.fail("Empty path should generate IOException");
-    } catch (IOException ex) {
+    } catch (IllegalArgumentException ex) {
```

Comment:
Previously it was throwing IOException when ClassRepository was trying to read file with empty string. Now it has checkArgument validation against such case.